src/aks-preview/HISTORY.md

```diff
@@ -6,6 +6,7 @@ Release History
 0.5.15
 +++++
 * Update to use 2021-05-01 api-version
+* Add argument '--enable-ultra-ssd'
```

Comment:
Please specify the feature of argument `--enable-ultra-ssd`, such as `Add argument '--enable-ultra-ssd' to enable UltraSSD on agent node pool`

Reply:
fixed